backend/tests/unit/services/test_history_service.py

```
@@ -945,11 +992,21 @@ async def test_cleanup_orphaned_sessions():
         if status_value in AlertSessionStatus.active_values():
             mock_active_sessions.append(session_dict)
     
-    # Mock repository responses
-    mock_repo.get_alert_sessions.return_value = {
-        "sessions": mock_active_sessions,
-        "total": len(mock_active_sessions)
-    }
+    # Mock repository responses - Phase 3: Repository now returns PaginatedSessions model
```

Ruff (0.12.2)

Line too long (89 > 88)

(E501)